libs/blocks/aside/aside.css

```diff
@@ -29,19 +35,29 @@
 }
 
 .aside.dark {
+  color: white;
   background-color: #1E1E1E;
 }
 
+.aside.light,
+.aside.split {
+  background-color: #f5f5f5;
```

Comment:
Shouldn't this be an author-able first row decorator and not set in style?

Reply:
I adjusted this a bit, but these were their final decisions on default light/dark colors. These are overrides by the first row authoring.